src/tools/rustdoc-gui/tester.js

```diff
@@ -210,12 +204,17 @@ async function main(argv) {
                 status_bar.erroneous();
                 failed = true;
             })
-        );
+            .finally(() => {
+                tests.splice(tests.indexOf(callback), 1);
```

Comment:
I would add a comment explaining that this removes the current test from the queue, I had to lookup what this does

Reply:
Sure!